### PR TITLE
chore(localdebug): add dotnet installation to tools/install driver

### PR DESCRIPTION
**Pull request by XiaofuHuang**
Add dotnet installation to `tools/install` driver

### CODE REVIEW

File: packages/fx-core/resource/package.nls.json [hunk 1]
```diff
@@ -1094,6 +1094,7 @@
   "driver.spfx.error.noValidAppCatelog": "There is no valid app catalog in your tenant. You can update the property 'createAppCatalogIfNotExist' in app.yml to true if you want us to create it for you or you can create it by yourself.",
   "driver.tools.error.invalidParameter": "Following parameter is missing or invalid for %s action: %s.",
   "driver.tools.error.funcInstallationError": "Failed to check and install Azure Functions Core Tools.",
+  "driver.tools.error.dotnetInstallationError": "Failed to check and install Dotnet.",
```
Comment (kuojianlu) [inline packages/fx-core/resource/package.nls.json:1097]: `.NET`?

**Review by kuojianlu — APPROVED** [review]

File: packages/fx-core/resource/package.nls.json (hunk 1 above)
Comment (XiaofuHuang) [reply packages/fx-core/resource/package.nls.json]: Updated, THX

File: packages/fx-core/src/component/driver/tools/installDriver.ts [hunk 2]
```diff
@@ -13,11 +14,13 @@ import { InvalidParameterUserError } from "./error/invalidParameterUserError";
 import { InstallToolArgs } from "./interfaces/InstallToolArgs";
 import { DepsManager, DepsType, EmptyLogger, EmptyTelemetry } from "../../../common/deps-checker";
 import { FuncInstallationUserError } from "./error/funcInstallationUserError";
+import { DotnetInstallationUserError } from "./error/dotnetInstallationUserError";
 
 const ACTION_NAME = "tools/install";
 const outputName = Object.freeze({
   SSL_CRT_FILE: "SSL_CRT_FILE",
   SSL_KEY_FILE: "SSL_KEY_FILE",
+  DOTNET_PATH: "DOTNET_PATH",
```
Comment (swatDong) [inline packages/fx-core/src/component/driver/tools/installDriver.ts:23]: Why output this? It's a bit weird there's only dotnet path but no func path.

Comment (XiaofuHuang) [reply packages/fx-core/src/component/driver/tools/installDriver.ts:23]: The dotnet/command driver need the path as input.

The func path is never used in the other drivers.

Comment (swatDong) [reply packages/fx-core/src/component/driver/tools/installDriver.ts:23]: Just to keep UX consistency - output `FUNC_PATH` as well when improving func install later.

**Review by swatDong — APPROVED** [review]

**Review by hund030 — APPROVED** [review]

**Pull request merged**